<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartTimeRangeDialog</class>
 <widget class="QDialog" name="ChartTimeRangeDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Changing time range for chart</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QDateTimeEdit" name="p_dateTimeEdit_StartRange"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="p_lbl_EndRange">
     <property name="text">
      <string>End range</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateTimeEdit" name="p_dateTimeEdit_EndRange"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="p_lbl_StartRange">
     <property name="text">
      <string>Start range</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChartTimeRangeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChartTimeRangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
